modules/heat_transfer/doc/content/source/materials/ElectromagneticHeatingMaterial.md

```diff
@@ -0,0 +1,37 @@
+# ElectromagneticHeatingMaterial
+
+!syntax description /Materials/ElectromagneticHeatingMaterial
+
+## Overview
+
+`ElectromagneticHeatingMaterial` provides the electric field and residuals of electromagnetic/electrostatic
+heating based objects as material properties. In particular, `ElectromagneticHeatingMaterial` provides the
+residuals for [ADJouleHeatingSource](ADJouleHeatingSource.md) and [JouleHeatingHeatGeneratedAux](JouleHeatingHeatGeneratedAux.md) within the Heat Transfer module.
+
+For the formulation of the electric field, users can declare:
+
+- ELECTROMAGNETIC: where the electric field can be supplied by the electromagnetic module
+- ELECTROSTATIC: where the user provides an electrostatic potential and the electric field ($E$) is defined as $E = \nabla V$,  where $V$ is the electrostatic potential
+
+For the formulation of the residuals, user can declare:
+
+- TIME DOMAIN: where the electric field is assumed to be derived from the time dependent Maxwell's equations
+- FREQUENCY DOMAIN: where the electric field is assumed to be derived from the frequency dependent Maxwell's equations. When using the frequency domain, the electric field is assumed to have a real and complex component.
+
```

Comment:
I'd like to see Latex for the equations that are used in the code in the documentation page. The documentation could be further improved by showing how the equations are affected by the different formulation tags, e.g. what in the material equation is reduced/ simplified by the different formulations?